app/resources/services/geoip.js

```diff
@@ -25,11 +25,21 @@ exports.get = function (req, res) {
         return
       }
 
+      // If ipstack returns an error, catch it and return a generic error.
+      // Log the error so we can examine it later.
+      // Do not use a falsy check here. A succesful response from ipstack does
+      // not contain the `success` property. It is only present when it fails.
+      if (body.success === false) {
```

Comment:
It looks like in order to access `body.success` you have to parse the data first. Without `JSON.parse(body)`, `body.success` is returning `undefined`.